app/models/user.rb

```diff
@@ -13,9 +13,11 @@ class User < ApplicationRecord
   has_many :friendships
   has_many :inverse_friendships, class_name: 'Friendship', foreign_key: 'friend_id'
 
-  # status = 1 --> Friends
-  # status = 0 --> Not Friends
-  # status = -1 --> Pending Friends
+  has_many :pending_friendships, -> { where(status: -1) }, class_name: 'Friendship', foreign_key: 'user_id'
+  has_many :pending_friends, through: :pending_friendships, source: :friend
```

Comment:
Yes, that is true, I originally implemented the project using boolean values, but I later changed it because I thought pending and rejected friend requests will have the same status, and the only way I saw to resolve it wast to delete the record, which I did not want to do. So using the integer values helps me maintain the record and just update where necessary.